com.unity.cinemachine/Samples~/Shared Assets/Prefabs/CapsuleCar.prefab

```diff
@@ -35,7 +35,7 @@ Transform:
   - {fileID: 4228984098192203911}
```

Comment:
This asset uses a mix of our input actions and the  old ones.  We should check everything and use our stuff only.